Source/Houdini/Houdini.cs

```diff
@@ -158,7 +158,7 @@ public override void UpdateImplementation(Implementation implementation)
       curImp = implementation;
     }
 
-    public override void UpdateOutcome(ProverInterface.Outcome o)
+    public override void UpdateOutcome(Outcome o)
```

Comment:
And this one could be renamed ProverOutcome for disambiguation with the other VcOutcome

Reply:
Made a fresh PR